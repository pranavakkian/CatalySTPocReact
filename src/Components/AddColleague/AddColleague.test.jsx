import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import AddColleaguePage from './index';

describe('AddColleaguePage', () => {
  const mockStore = configureMockStore();

  test('Component rendered successfully', () => {
    const store = mockStore(() => ({
      UserInfoReducer: {
        isLoggedIn: true,
        userName: 'test',
      },
    }));
    const { container } = render(<Provider store={store}><AddColleaguePage /></Provider>);
    expect(container).toMatchSnapshot();
  });
});
