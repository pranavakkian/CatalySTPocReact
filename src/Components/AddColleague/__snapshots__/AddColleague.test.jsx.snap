// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddColleaguePage Component rendered successfully 1`] = `
<div>
  <div>
    <h1
      class="ui center aligned header"
    >
      Add Colleague
    </h1>
  </div>
</div>
`;
